### ISSUE TITLE
Feature Request: More Width for Custom Index on Homepage

### ISSUE THREAD

**Issue by saurabh-mish** [imfing/hextra#29]
If we're using a custom `_index.md` in the `content/` directory, the width of the text is too narrow.

In my opinion it makes sense for this text to span a little wider (10% - 15% more), because there are no navigation menus or either side.

**Comment by imfing** [comment]
Thanks for the feedback, I was also considering adding a global flag for setting the width of the page.

**Comment by saurabh-mish** [comment]
Is is possible to use `layout: hextra-home` frontmatter in `content/_index.md` for custom content?

**Comment by imfing** [comment]
> I'm just looking to have a better homepage :)

If you don't mind overriding the layout, you can copy the [`layouts/index.html`](https://github.com/imfing/hextra/blob/main/layouts/index.html) and modify the second line:

https://github.com/imfing/hextra/blob/171399889dc0b300f8bb0b0f7ecf1c424533ef74/layouts/index.html#L2

to:


```diff
-  <div class="mx-auto flex max-w-screen-xl">
+  <div class="mx-auto flex max-w-[90rem]">
```

which should make it a bit wider. This may be the best option for this case.

**Comment by imfing** [comment]
Never mind, I think it makes more sense to make the `text-only` index page wider. Fixed in #30

**Comment by saurabh-mish** [comment]
Thanks Xin !
